test/func/tests/analytics/index.testplane.js

```diff
@@ -0,0 +1,117 @@
+const childProcess = require('child_process');
+const path = require('path');
+const {PORTS} = require('../../utils/constants');
+
+const projectDir = path.resolve(__dirname, '../../fixtures/analytics');
+
+const generateFixtureReport = async (args, env) => {
+    await new Promise(resolve => {
+        const proc = childProcess.spawn('npx', ['testplane', ...args], {cwd: projectDir, env: {...process.env, ...env}});
+
+        proc.on('exit', () => {
+            resolve();
+        });
+    });
+};
+
+const launchStaticServer = () => {
+    return new Promise((resolve) => {
+        const proc = childProcess.spawn('node', [path.resolve(__dirname, '../static-server.js')], {
+            stdio: ['pipe', 'pipe', 'pipe', 'ipc'],
+            env: {
+                ...process.env,
+                STATIC_DIR: path.resolve(__dirname, '../..'),
+                PORT: PORTS.analytics.server
+            }
+        });
+
+        proc.on('message', () => {
+            resolve(proc);
+        });
+    });
+};
+
+describe('Analytics', () => {
+    let server;
+
+    afterEach(() => {
+        server?.kill();
+    });
+
+    it('should include metrika script by default', async ({browser}) => {
+        await generateFixtureReport(['-c', 'enabled.testplane.conf.js']);
+        server = await launchStaticServer();
+
+        await browser.url(browser.options.baseUrl.replace('index.html', 'new-ui.html'));
+        const scriptElement = await browser.$('div[data-qa="metrika-script"] script');
+
+        await expect(scriptElement).toBeExisting();
+    });
+
+    it('should track feature usage when opening info panel', async ({browser}) => {
+        await generateFixtureReport(['-c', 'enabled.testplane.conf.js']);
+        server = await launchStaticServer();
+
+        await browser.url(browser.options.baseUrl.replace('index.html', 'new-ui.html'));
+        await browser.execute(() => {
+            window.ym = (...args) => {
+                if (!window.ym.calls) {
+                    window.ym.calls = [];
+                }
+                window.ym.calls.push(args);
+            };
+        });
+
+        await browser.$('[data-qa="footer-item-info"]').click();
+
+        const [metrikaCall] = await browser.execute(() => {
+            return window.ym.calls;
+        });
+
+        expect(metrikaCall[1]).toEqual('reachGoal');
+        expect(metrikaCall[2]).toEqual('FEATURE_USAGE');
+        expect(metrikaCall[3]).toEqual({featureName: 'Open info panel'});
```

Comment:
not critical: This check looks fragile and may break if this click would produce other metrika action before this one. Maybe we should save all `metricaCalls` and check if it has any call, matching args `['reachGoal', 'FEATURE_USAGE', {featureName: 'Open info panel'}]`?